library/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -30,6 +37,34 @@ public object Chucker {
             .setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
     }
 
+    /**
+     * Create a shortcut to launch Chucker UI.
+     * @param context An Android [Context].
+     */
+    internal fun createShortcut(context: Context) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
+            run outside@{
+                context.getSystemService(ShortcutManager::class.java)?.let { sm ->
+                    sm.dynamicShortcuts.forEach {
+                        if (it.id == SHORTCUT_ID) return@outside
```

Comment:
No need for `run` with an outside label. You can return here with `return@let`.

Reply:
Your are right